python/ray/autoscaler/sdk.py

```diff
@@ -62,8 +62,9 @@ def run_on_cluster(cluster_config: Union[dict, str],
                    cmd: Optional[str] = None,
                    run_env: str = "auto",
                    no_config_cache: bool = False,
-                   port_forward: Union[int, List[int]] = None,
-                   with_output: bool = False) -> str:
+                   port_forward: Union[Tuple[int, int], List[Tuple[int, int]],
+                                       None] = None,
```

Comment:
It might be useful to have something like
`PortForward = Union[Tuple[int, int], List[Tuple[int, int]], None]`
and then use `PortForward` as the type for `port_forward`.

Reply:
I agree, this is quite wordy.